lib/easeljs.js

```diff
@@ -6696,7 +6696,7 @@ this.createjs = this.createjs||{};
 	 * @return {String} The image data url for the cache.
 	 **/
 	p.getCacheDataURL = function() {
-		return this.bitmapCache?this.bitmapCache.getDataURL():null;

```

Comment:
this is a typo. the correct function name is getCacheDataURL

https://github.com/CreateJS/EaselJS/blob/master/src/easeljs/filters/BitmapCache.js#L516